tests/src/stateless_validator.rs

```diff
@@ -18,41 +18,77 @@ mod tests {
 
     #[tokio::test]
     async fn prove_empty_block() {
-        empty_block(Action::Prove).await;
+        let el_zkvms = filter_el_zkvm_pairs_from_env(vec![
+            (ExecutionClient::Reth, ErezkVM::SP1),
+            (ExecutionClient::Reth, ErezkVM::Risc0),
+            (ExecutionClient::Reth, ErezkVM::OpenVM),
+            // (ExecutionClient::Reth, ErezkVM::Pico), // See https://github.com/eth-act/ere/issues/173
+            (ExecutionClient::Ethrex, ErezkVM::SP1),
+            // (ExecutionClient::Ethrex, ErezkVM::Risc0), // See https://github.com/eth-act/ere/issues/121
+            // (ExecutionClient::Ethrex, ErezkVM::OpenVM), // See https://github.com/eth-act/ere/issues/168
+            // (ExecutionClient::Ethrex, ErezkVM::Pico), // See https://github.com/eth-act/ere/issues/174
+        ]);
+        empty_block(Action::Prove, &el_zkvms).await;
     }
 
     #[tokio::test]
     async fn execute_empty_block() {
-        empty_block(Action::Execute).await;
+        let el_zkvms = filter_el_zkvm_pairs_from_env(vec![
+            (ExecutionClient::Reth, ErezkVM::SP1),
+            (ExecutionClient::Reth, ErezkVM::Risc0),
+            (ExecutionClient::Reth, ErezkVM::OpenVM),
+            (ExecutionClient::Reth, ErezkVM::Pico),
+            (ExecutionClient::Ethrex, ErezkVM::SP1),
+            // (ExecutionClient::Ethrex, ErezkVM::Risc0), // See https://github.com/eth-act/ere/issues/121
+            // (ExecutionClient::Ethrex, ErezkVM::OpenVM), // See https://github.com/eth-act/ere/issues/168
+            // (ExecutionClient::Ethrex, ErezkVM::Pico), // See https://github.com/eth-act/ere/issues/174
+        ]);
+        empty_block(Action::Execute, &el_zkvms).await;
     }
 
     #[tokio::test]
     async fn execute_mainnet_blocks() {
```

Comment:
Instaed of doing a single `run_guest` with all the zkvms, now we iterate on each and run independently.

This is requried since now Pico needs to do some exceptions to avoid running specific tests that we must run in others... so the json files aren't the same set for all zkvms anymore (thus why we run independently now).